crates/flox-rust-sdk/src/providers/git.rs

```diff
@@ -340,6 +340,125 @@ impl GitCommandProvider {
             path,
         })
     }
+
```

Comment:
Do we want somewhat specific helpers to live in the git module?

Reply:
Some of these are a bit targeted at operations we'll need for floxmeta - I dunno if they'll be generally useful. So I was wondering if we should leave them in the git module or put them in the floxmeta one

Reply:
I say leave them in, we can always move them later if we care